client/remote.cpp

```diff
@@ -410,7 +412,13 @@ static int build_remote_int(CompileJob &job, UseCSMsg *usecs, MsgChannel *local_
                 throw client_error(4, "Error 4 - unable to stat version file");
             }
 
-            EnvTransferMsg msg(job.targetPlatform(), job.environmentVersion());
+            int extract_priority = getpriority(PRIO_PROCESS, 0);
+            if (-1 == extract_priority){
+                log_perror("Unable to determine process priority");
+                extract_priority = 0;
+            }
+
+            EnvTransferMsg msg(job.targetPlatform(), job.environmentVersion(), extract_priority);
```

Comment:
Also, compile priority is already controlled by the daemon, I think that is the value you use be using.